src/appleseed/foundation/image/genericimagefilewriter.h

```diff
@@ -92,7 +93,12 @@ class APPLESEED_DLLSYMBOL GenericImageFileWriter
     struct Impl;
     Impl* impl;
 
+#if OIIO_VERSION >= 20000
+    std::unique_ptr<OIIO::ImageOutput> m_writer;
```

Comment:
You need to include `<memory>` for the definition of `std::unique_ptr<>`.

Reply:
Also, can we move `m_writer` and `m_filename` to the private impl?

Reply:
Done, but still testing.